test/services/query-builder.test.js

```diff
@@ -70,20 +71,82 @@ describe('services > query-builder', () => {
 
         expect(includes).toStrictEqual([{
           as: 'userAccessor',
-          attributes: ['uid', 'name'],
+          attributes: ['p_id', 'uid', 'name'],
           model: { name: 'user' },
         }]);
       });
 
-      it('should always include the target key even if not specified', () => {
+      it('should always include the source key and primary key even if not specified', () => {
         expect.assertions(1);
         const { model, queryBuilder } = setup();
 
         const includes = queryBuilder.getIncludes(model, ['user.name', 'user.id', 'user.badField']);
 
         expect(includes).toStrictEqual([{
           as: 'userAccessor',
-          attributes: ['uid', 'name'],
+          attributes: ['p_id', 'uid', 'name'],
```

Comment:
Why do we need the source key?
The primary key + display field should be enough right?
Is it for join purpose?